test/lib/utils/createPoolHelper.ts

```diff
@@ -32,6 +33,7 @@ export async function doCreatePool(
             [PoolType.StableSurge]: stableSurgeFactoryAbiExtended,
             [PoolType.GyroE]: gyroECLPPoolFactoryAbiExtended,
             [PoolType.ReClamm]: reClammPoolFactoryAbiExtended,
+            [PoolType.LiquidityBootstrapping]: lBPoolFactoryAbi_V3,
```

Comment:
can change to extended ABI to match others.

Reply:
addressed